.github/workflows/ci.yml

```
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    type: [opened, synchronize]
+
+jobs:
+  test:
+    name: Test Node.js ${{ matrix.node-version }} on ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [12.x, 14.x, 16.x]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'yarn'
+    - name: install dependencies
+      run: yarn install
```

```suggestion
    - run: yarn install
```

Lets either use names for all or names for none.